src/components/ListingCard.js

```diff
@@ -1,24 +1,52 @@
-import React from "react";
+import React, { useState } from "react"
+
+function ListingCard({
+  listing: { id, description, image, location },
+  onDelete,
+}) {
+  const [favorite, setFavorite] = useState(false)
+
+  function handleFavoriteClick() {
+    setFavorite(!favorite)
+  }
+
+  function handleDeleteClick() {
+    fetch(`http://localhost:6001/listings/${id}`, {
+      method: "DELETE",
+    }).then((response) => response.json())
+    onDelete(id)
+  }
```

Comment:
You aren't doing anything with the response so you can remove that piece of code. Also, the `onDelete` should happen in a `then`:
```suggestion
  function handleDeleteClick() {
    fetch(`http://localhost:6001/listings/${id}`, {
      method: "DELETE",
    }).then(() => onDelete(id))
  }
```